priv/repo/migrations/20160818132546_add_organization_membership.exs

```diff
@@ -0,0 +1,20 @@
+defmodule CodeCorps.Repo.Migrations.CreateOrganizationMembership do
+  use Ecto.Migration
+
+  def change do
+    create table(:organization_memberships) do
+      add :role, :string, null: false
+      add :organization_id, references(:organizations, on_delete: :nothing), null: false
+      add :member_id, references(:users, on_delete: :nothing), null: false
+
+      timestamps()
+    end
+
+    create index(
+      :organization_memberships,
+      [:member_id, :organization_id],
+      name: :organization_memberships_member_id_index,
```

Comment:
Is it required? I thought you don't need to be explicit with the name on indexes?